### PR TITLE
Documents for the 3-level Hamiltonian and quantum gates

### PR DESCRIPTION
**Pull request by ChenZhao44**
This PR includes some documents for the 3-level support.

### CODE REVIEW

**Comment by Roger-luo** [comment]
the docstirng looks good to me, @Wang-Shengtao got some time to review the content in markdown?

File: docs/src/3-level.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+i \hbar \dfrac{\partial}{\partial t} | \psi \rangle = \hat{\mathcal{H}}(t) | \psi \rangle,  \\
+
+\begin{aligned} 
+    \frac{\mathcal{H}(t)}{\hbar} = & \sum_i \frac{\Omega_i^{\mathrm{hf}}(t)}{2}\left(e^{i \phi_i^{\mathrm{hf}}(t)}|0_i\rangle\langle 1_i|+e^{-i \phi_i^{\mathrm{hf}}(t)}| 1_i\rangle\langle 0_i|\right)-\sum_i \Delta_i^{\mathrm{hf}}(t)|1_i\rangle\langle 1_i| \\ 
+    & +\sum_i \frac{\Omega_i^{\mathrm{r}}(t)}{2}\left(e^{i \phi_i^{\mathrm{r}}(t)}|1_i\rangle\langle r_i|+e^{-i \phi_i^{\mathrm{r}}(t)}| r_i\rangle\langle 1_i|\right)-\sum_i\left[\Delta_i^{\mathrm{hf}}(t)+\Delta_i^{\mathrm{r}}(t)\right]\left|r_i\right\rangle\left\langle r_i\right| \\ 
+    & +\sum_{i<j} V_{i j}\left|r_i\right\rangle\left\langle r_i|\otimes| r_j\right\rangle\left\langle r_j\right| .
+\end{aligned}
+```
+
+Here, $|0\rangle$ and $|1\rangle$ represent two different hyperfine levels
```
Comment (Roger-luo) [inline docs/src/3-level.md:21]: change the math mark to Julia flavor to avoid conflict

**Comment by Roger-luo** [comment]
can you try merge master to see if the CI passes?

**Comment by ChenZhao44** [comment]
> can you try merge master to see if the CI passes?

Seems good now.

**Review by Roger-luo — APPROVED** [review]

**Pull request merged**